src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Runtimes/MissingSdkRuntimeDetector.cs

```diff
@@ -4,6 +4,11 @@
 
 namespace Microsoft.VisualStudio.ProjectSystem
 {
+    /// <summary>
+    /// Detect during solution load the Net Core runtime version based on the target framework 
+    /// used in a project file and pass this version to a service that can install the
+    /// runtime component if not installed.
+    /// </summary>
     [Export(ExportContractNames.Scopes.ConfiguredProject, typeof(IProjectDynamicLoadComponent))]
     [AppliesTo(ProjectCapability.DotNet)]
     internal class MissingSdkRuntimeDetector : OnceInitializedOnceDisposedAsync, IProjectDynamicLoadComponent
```

Comment:
The `_isEnabled` field in this class is redundant. You can replace it with `IsInitialized` from the base class.

Reply:
changed

Reply:
Your update only removes one read of the field. You should remove the field itself too.